internal/pkg/daemon/selinuxprofile/selinuxprofile.go

```diff
@@ -67,8 +76,23 @@ func (sph *selinuxProfileHandler) Init(
 	if sph.cli == nil {
 		sph.cli = cli
 	}
-	err := sph.cli.Get(ctx, key, sph.sp)
-	return err
+	// initiate the SelinuxProfile object
+	if err := sph.cli.Get(ctx, key, sph.sp); err != nil {
+		return err
+	}
+
+	// Matches alpha numerical names in upper and lower-case, as well as
+	// dashes and underscores and the @ for our @self usage.
```

Comment:
If you end up pushing a new version, please fix the comment because now we match for self directly.